include/boost/geometry/algorithms/detail/convex_hull/graham_andrew.hpp

```diff
@@ -225,6 +224,7 @@ class graham_andrew
             point_type const& left, point_type const& right,
             SideStrategy const& side)
     {
+        //std::cout << "add to hull" << std::endl;
```

Comment:
Is this a left over?

Reply:
Yes, will be removed (together with a few other comments).